### PR TITLE
feat: create custom application menu

### PR DESCRIPTION
**Pull request by josepholiveira**
In this PR, I created a custom application menu with some useful actions, like the action to create
a New Connection.

### CODE REVIEW

File: src/screen/ConnectionsList/index.tsx [hunk 1]
```diff
@@ -12,6 +13,12 @@ const ConnectionsList: React.FC = () => {
   const [connections] = useRecoilState(connectionsState)
   const [isCreateModalOpen, toggleCreateModalOpen] = useToggle(false)
 
+  useEffect(() => {
+    ipcRenderer.on('newConnection', function () {
```
Comment (diego3g) [inline src/screen/ConnectionsList/index.tsx:17]: Maybe we should return a function from this useEffect to unregister the ipcRenderer event listener.

**Comment by diego3g** [comment]
Can you apply the i18n messages as we merged #57?

**Comment by josepholiveira** [comment]
> Can you apply the i18n messages as we merged #57?

Just did it, waiting for reviews <3

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib"
```
Comment (diego3g) [inline .vscode/settings.json:2]: I think we should keep this out of the repo.

Comment (josepholiveira) [reply .vscode/settings.json]: Just removed it, should be ok now!

**Pull request merged**